.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,124 @@
+name: 🐛 Bug report
+description: Create a report to help us improve.
+labels:
+  - Bug
+
+body:
+  - type: textarea
```

Comment:
```suggestion
  - type: markdown
```

Reply:
I also first thought of `markdown` when configuring this, but the [docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#markdown) state it's only used for display:

> You can use a markdown element to display Markdown in your form that provides extra context to the user, but is not submitted.

Actually, when you click on a `textarea` input, it transforms into a markdown input:


https://user-images.githubusercontent.com/16200219/129625604-e5f96d7c-53c4-40f6-86d3-d311feb0f320.mp4

See the demo at https://github.com/pierlon/amp-wp/issues/new?assignees=&labels=Bug&template=bug_report.yml

Reply:
Oh, I had assumed that `textarea` was for unformatted text.